pom.xml

```diff
@@ -212,6 +212,7 @@
     <pitest.plugin.timeout.constant>50000</pitest.plugin.timeout.constant>
     <pitest.plugin.threads>4</pitest.plugin.threads>
     <sonar.test.exclusions>**/test/resources/**/*,**/it/resources/**/*</sonar.test.exclusions>
+    <argLine>-Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8</argLine>
```

Comment:
we spend some much time to make localized builds, I am not ok to make `-Duser.language=en -Duser.country=US` so easily.
I am ok `-Dfile.encoding=UTF-8` 

I might be ok with extra maven profiles to activate on windows only when encoding is not UTF-8.

Reply:
@romani issue created identified the underlying issue, which is an issue with `OutputStreamWriter` and I think we need to convert it to `PrintWriter`.

Reply:
if problems in code, it should be fixed in code, but not in configuration for all OSs.

Reply:
I agree which is why I dug up the underlying issue. I can fix up the issue to make it clearer.